### PR TITLE
[port_buffer_drop] Add Port buffer drop counters in SONiC doc

### PR DESCRIPTION
**Pull request by mykolaf**
Add req doc for https://github.com/Azure/sonic-swss/pull/1368 as the community requested

Signed-off-by: Mykola Faryma <mykolaf@mellanox.com>

### CODE REVIEW

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+The solution for now is to introduce a new FC group with a larger polling interval than the rest of port counters.
+Also limit user configuring a small interval for this FC group via CLI.
+
+# 2 Requirements
+
+## 2.1 Functional Requirements
+1. New flex counter group is introduced for the port-level buffer drop counters
+2. The FC group is enabled by default
+3. The polling interval is 60s by defaul
```
Comment (liat-grozovik) [inline doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md:55]: /s defaul default

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## 3.1 CLI (and usage example)
+
+### 3.1.1 Displaying the FC configuration
+```
+admin@sonic:~$ counterpoll show
+Type                        Interval (in ms)    Status
+--------------------------  ------------------  --------
+QUEUE_STAT                  default (10000)     enable
+PORT_STAT                   default (1000)      enable
+PORT_BUFFER_DROP            default (60000)     enable
```
Comment (liat-grozovik) [inline doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md:77]: suggest to mark the added one comparing to the current implementation.

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+### 3.1.2 Enabling/Disabling the polling 
+```
+admin@sonic:~$ counterpoll port-buffer-drop enable
+admin@sonic:~$ counterpoll port-buffer-drop disable
+```
+
+### 3.1.3 Setting new polling interval 
+```
+admin@sonic:~$ counterpoll port-buffer-drop interval 30000
+```
```
Comment (liat-grozovik) [inline doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md:94]: do you have schema update, right? please add it to the design.

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md (hunk 1 above)
Comment (mykolaf) [reply doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md]: Done

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md (hunk 2 above)
Comment (mykolaf) [reply doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md:95]: Done

File: doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md (hunk 3 above)
Comment (mykolaf) [reply doc/port_buffer_drop_counters/sonic_port_buffer_drop_counters.md:112]: Port buffer drop counters does not introduce any new schema.

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**